repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
+          "default": ["slave_public"],
```

Comment:
Not sure I agree, unless the admin-router is offering some kind of authentication.
I'd rather be able to use the public ip in our docs, because it makes things like retrieving container logs possible.